src/main/scala/mesosphere/marathon/state/PathId.scala

```diff
@@ -180,4 +180,6 @@ object PathId {
     * Needed for AppDefinitionValidatorTest.testSchemaLessStrictForId.
     */
   val absolutePathValidator = isTrue[PathId]("Path needs to be absolute") { _.absolute }
+
+  val topLevel = isTrue[PathId]("Path needs to be top-level") { _.parent.isRoot }
```

Comment:
I understand the wording for "topLevel", as this is used in validations like "xxx is TopLevel", maybe we can make it more explicit like "validTopLevelGroup"? On my first encounter I thought it'd return the topLevel group of that path...

Reply:
(On a side note, why don't we add a method for Path: `.isTopLevel `)